manifests/install.pp

```diff
@@ -8,4 +8,17 @@
   package{'ferm':
     ensure => 'latest',
   }
+
+  case $facts['os']['family'] {
+    'RedHat': {
+      if scanf($facts['os']['release']['major'], '%i')[0] <= 6 {
+        file{'/etc/init.d/ferm':
```

Comment:
could you create a new parameter like `$manage_initfile` or something like that? The file isn't in the official rpm, so people might already have a custom script inplace that we overwrite. So we should only deploy the file, if a user explicitly requests it. Otherwise the next release needs to be a major one instead a minor one.